### ISSUE TITLE
Cannot connect via USB to C100X16E

### ISSUE THREAD

**Issue by dirz88** [fquinto/bticinoClasse300x#36]
Hi all
I've builded the firmware and failing to flash it to my device. The software MyHOME simply doesn't recognize it. It fails with the error "Unable to contact device". 

I've also tried it like suggested here: https://github.com/fquinto/bticinoClasse300x/issues/31#issuecomment-2143834566

Could anyone give me a hint to solve this?
Thanks a lot.

**Comment by sstidl** [comment]
Did you connect the USB cable after starting to flash?

I did use a fresh install of win11 without any additional security software. It didn't work on a laptop with trendmicro antivirus installed.

**Comment by dirz88** [comment]
Yep - I‘ve connected it after starting to flash. I‘ve tried 3 notebooks with WIN10, WIN11 with fully disabled Antivirus. I also rebooted the Classe by unplug the BUS, unfortunately that didn‘t help.

I‘ve also tried 3 different USB cables. According to the iOS App ‚Securty‘ the Classe does have version 2.03.01 installed. I‘ll have a look if I can try to get a PC with plain OS installed. Thanks for helping so far.

**Comment by sstidl** [comment]
I don't know if a device running 2.x will work. The project here only supports 1.x firmware devices. Never heard of anyone downgrading so far.

**Comment by tarbax** [comment]
Any succes with firmware version 2.03.01 ? 
I also have that firmware...

**Comment by sstidl** [comment]
As stated in other discussions, firmware 2.x is not supported by this repo

**Comment by majorhector** [comment]
If you have 2.x firmware you can use btcino_intercom integration in HACS without flashing anything :)

https://github.com/k-the-hidden-hero/bticino_intercom
